Terminal.Gui/ConsoleDrivers/NetDriver.cs

```diff
@@ -1685,7 +1683,7 @@ bool IMainLoopDriver.EventsPending ()
 
         if (!_eventReadyTokenSource.IsCancellationRequested)
         {
-            return (_resultQueue.Count > 0) || _mainLoop.CheckTimersAndIdleHandlers (out _);
+            return _resultQueue.Count > 0 || _mainLoop.CheckTimersAndIdleHandlers (out _);
```

Comment:
If possible, we should provide an override without the out param. Discards still result in a wrapper method and passing the parameter around. Minimal impact if only reference types. Potentially tangible impact if value types can be here.